libcxx/include/__type_traits/desugars_to.h

```diff
@@ -46,6 +52,10 @@ struct __totally_ordered_less_tag {};
 template <class _CanonicalTag, class _Operation, class... _Args>
 inline const bool __desugars_to_v = false;
 
+template <class _Operation, class... _Args>
+inline const bool __desugars_to_v<__less_tag, _Operation, _Args...> =
+    __desugars_to_v<__totally_ordered_less_tag, _Operation, _Args...>;
```

Comment:
I think I would remove this since it's effectively not used. In the future, if we want to create a real "hierarchy" of such tags, then it would make sense to re-add it.